test/ng/directive/inputSpec.js

```diff
@@ -3046,6 +3046,26 @@ describe('input', function() {
       expect(inputElm[0].getAttribute('value')).toBe('something');
     });
 
+    they('should update the $prop "value" property and attribute after change the "value" property', {
+      input: '<input type="text" ng-value="value">',
+      textarea: '<textarea ng-value="value">></textarea>'
+    }, function(tmpl) {
+      var element = helper.compileInput(tmpl);
+
+      $rootScope.$apply(function() {
+        $rootScope.value = 'something';
+      });
+      expect(element[0].value).toBe('something');
+      expect(element[0].getAttribute('value')).toBe('something');
+
+      helper.changeInputValueTo('newValue');
```

Comment:
This is not 'redundant' it important to test that the 
"input binding to the model" is still working after user edited the input, without it we are not testing the Issue 
if "value property"  has no value, it will read the value from "value attribute" 
The Issue happened when user type something in the input,
see the following step
1. User edit the input
2 . then change the model value programmatically by changing the scope.
3. The input is not updated will the new value of the model

I am trying to simulating what happen in the Issue https://github.com/angular/angular.js/issues/13984

do i need to add expect after it ?